website/content/docs/concepts/domain-model/index.mdx

```diff
@@ -52,7 +52,26 @@ The diagram below illustrates the relationship between the different IAM compone
 
 ## Target and Host Resources
 
-Boundary exposes endpoints to users as [Targets](/docs/concepts/domain-model/targets). A target is a resource representing a networked service, with an associated set of permissions, that allows a user to connect and interact with Boundary within a single  session. A target may contain references to [Host](/docs/concepts/domain-model/hosts) sources and [Credential](/docs/concepts/domain-model/credentials) sources. Users who access a target create an authorized session to one of its host sources with returned credentials from the target’s credential source.
+Boundary exposes endpoints to users as [Targets](/docs/concepts/domain-model/targets). A target is a resource representing a networked service, with an associated set of permissions, that allows a user to connect and interact with Boundary within a single session. A target may contain references to [Host](/docs/concepts/domain-model/hosts) sources or an address, and [Credential](/docs/concepts/domain-model/credentials) sources. Users who access a target create an authorized session to its address or one of its host sources with returned credentials from the target’s credential source.
+
+A target exposes two distinct and mutually exclusive configuration paths for its
+representation of a networked service:
+
+- Target with an address
+- Target with host sources
+
+A target with an address represents a target that is set-up with a single IP
+address or DNS name directly on the target resource. This mechanism provides
+benefits for new Boundary users, as it's less complex to initially set-up and
+understand, or users who don't require Boundary's flexibility and scalability.
+Because of its simplicity, using this path on environemnts where a significant
```

Comment:
```suggestion
Because of its simplicity, using this path on environments where a significant
```

Reply:
Haha thanks for spotting, fixed with [c628d10](https://github.com/hashicorp/boundary/pull/2708/commits/c628d10cd187f8afaa470911c764051a56b7f4fa)